templates/careers/departments.html

```diff
@@ -19,76 +19,242 @@ <h1 class="p-heading--2">Departments at Canonical</h1>
 </section>
 
 <section class="p-strip is-bordered">
-  <div class="row">
-    <div class="col-4 u-sv3">
-      <h2>Engineering</h2>
-      <p>Open source is transforming the entire stack. This is your chance to be right at the centre of that revolution, to shape the platforms and tools that millions use to invent our global technology future. From bare metal to cloud and high performance computing, from AI and big data to the web and connected devices, open source is the key ingredient for success.</p>
-      <a href="/careers/all?filter=Engineering">Check out open Engineering roles</a>
+  <div class="p-strip is-shallow">
+    <div class="row">
+      <hr>
+      <div class="col-6">
+        <h2>Engineering</h2>
+        <p>Open source is transforming the entire stack. This is your chance to be right at the centre of that revolution, to shape the platforms and tools that millions use to invent our global technology future. From bare metal to cloud and high performance computing, from AI and big data to the web and connected devices, open source is the key ingredient for success.</p>
+        <a href="/careers/all?filter=Engineering">Check out open Engineering roles</a>
+      </div>
+      <div class="col-6 u-vertically-center u-hide--small u-hide--medium">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/379b19b9-department-engineering-%402x.jpg",
+          alt="",
+          width="2216",
+          height="834",
```

Comment:
The ratio of the images seems to be approximately 1.9:1 in Figma. Compared to design, the height looks a bit short.